models/nft/arbitrum/nft_arbitrum_transfers.sql

```diff
@@ -0,0 +1,74 @@
+ {{ config( alias='transfers') }}
```

Comment:
This would be the last config block that need to add incremental strategy in this PR.

Reply:
ah well spotted I had missed that one, just fixed it with a new commit :)